includes/class-coblocks-font-loader.php

```diff
@@ -37,16 +37,16 @@ public static function register() {
 	/**
 	 * The Plugin version.
 	 *
-	 * @var string $_version
+	 * @var string $version
 	 */
-	private $_version;
+	private $version;
 
 
 	/**
 	 * The Constructor.
 	 */
 	public function __construct() {
-		$this->_version = COBLOCKS_VERSION;
+		$this->version = COBLOCKS_VERSION;
```

Comment:
Is this needed? Can we just reference `COBLOCKS_VERSION` throughout?

Reply:
Good catch. Didn’t think to look into how it was used, just removed the unnecessary define abstraction.

Reply:
Fixed.